<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryBuilderTool</class>
 <widget class="QMainWindow" name="QueryBuilderTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Query Builder Tool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/builder.png</normaloff>icons/builder.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="ComplexQuery">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Complex Query Builder</string>
    </property>
    <widget class="QPushButton" name="addComplexButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add to Queries</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cqRadioUnion">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Union</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cqRadioIntersection">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Intersection</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cqBoxLeft">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cqBoxRight">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="cqRadioComplement">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Complement</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="QueryList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Query List</string>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>30</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Coming soon!</string>
     </property>
     <property name="text">
      <string>Edit Existing Query</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeQueryButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>70</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Entire row must be selected.</string>
     </property>
     <property name="text">
      <string>Remove Query</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>601</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="hideRowButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>110</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Entire row must be selected.</string>
     </property>
     <property name="text">
      <string>Hide Row(s)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showRowsButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>150</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show All Rows</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearQueriesButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>190</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear All Queries</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SimpleQuery">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>451</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Simple Query Builder</string>
    </property>
    <widget class="QCheckBox" name="caseSenseCheck">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Case Sensitive</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="wholeWordCheck">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Whole Word</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="specDelimsCheck">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Allow Special Delimeters ( , -   )</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="pluralsCheck">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>100</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Plurals</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="constituentsCheck">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Constituents</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addSimpleButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add to Queries</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="phraseEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Type query word/phrase here.</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSaveQL"/>
    <addaction name="actionSaveAsQL"/>
    <addaction name="actionLoadQL"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionExportST"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionSaveQL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoadQL">
   <property name="icon">
    <iconset>
     <normaloff>icons/open.png</normaloff>icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Load Query File</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset>
     <normaloff>icons/refresh.png</normaloff>icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionSaveAsQL">
   <property name="icon">
    <iconset>
     <normaloff>icons/save.png</normaloff>icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExportST">
   <property name="text">
    <string>Export to Summary Tool</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
